plugins/ptp/libptp/usb.c

```diff
@@ -164,6 +164,7 @@ ptp_usb_senddata (PTPParams* params, PTPContainer* ptp,
 	res = ptp_write_func((char*)&usbdata, wlen, params->data);
 	if (res != wlen) 
 	{
+		ret = PTP_ERROR_IO;
 		ptp_usb_error(params, "ptp2/usb_senddata : request code 0x%04x sending data error 0x%04x",ptp->Code,ret);
```

Comment:
```suggestion
		ptp_usb_error(params, "ptp2/usb_senddata : request code 0x%04x sending data error 0x%04x",ptp->Code,PTP_ERROR_IO);
```
This is a little bit simpler. :)